### ISSUE TITLE
Enable AutoReboot Policy

### ISSUE THREAD

**Issue by gkeishin** [openbmc/openbmc#2056]
This attribute value must be default to `True`

https://github.com/openbmc/meta-phosphor/blob/da48267fd87ab8aebb862a731b6669af279e81ab/common/recipes-phosphor/settings/phosphor-settings-defaults/defaults.yaml#L4

```
/xyz/openbmc_project/control/host0/auto_reboot:
    Interface: xyz.openbmc_project.Control.Boot.RebootPolicy
    Defaults:
AutoReboot: 'false'
```

to `true`

**Comment by gkeishin** [comment]
https://gerrit.openbmc-project.xyz/#/c/5669/

**Comment by rfrandse** [comment]
<https://gerrit.openbmc-project.xyz/5669> Enable "AutoReboot" property by default
Resolves   #2056 Enable AutoReboot Policy
